internal/test/testdata/engine/case_30.yaml

```diff
@@ -0,0 +1,111 @@
+# yaml-language-server: $schema=../.jsonschema/EngineTestCase.schema.json
+---
+description: "Resource policy with REQUIRE_PARENTAL_CONSENT_FOR_ALLOWS, leaf skipped"
+inputs:
+  [
+    {
+      "requestId": "test",
+      "actions": [
+        "move", # allowed for admin in the parent scope, allow
+        "acme_action", # allowed for employee in the parent scope, deny
```

Comment:
This one's also a duplicate of the `move` case (albeit for another role). It can probably be removed too?